2022/assets/css/talk-list.css

```diff
@@ -0,0 +1,60 @@
+.talk-list--item {
+    margin-bottom: 25px;
+    border: 1px solid #d5d5d5;
+    padding: 16px;
+    border-radius: 5px;
+}
+
+.talk-item-preheader {
+    padding-bottom: 8px;
+}
+
+.talk-item-preheader > div {
+    display: inline-block;
+    margin-left: 5px;
+    margin-right: 5px;
+}
+
+.talk-item-header {
+    padding-bottom: 7px;
+}
+
+.talk-item-header h3 {
+    margin: 0;
+    display: inline;
+    color: inherit;
+}
+
+.talk-language .fa,
+.talk-level .fa {
+    margin-right: 5px;
+}
+
+.talk-language-flag {
```

Comment:
No idea of CSS, but is this and the other `.talk-language-flag.*` used? At least I haven't seen any flag in the preview.

Reply:
You're correct, is a leftover from an idea of using a flag for the language.